core-bundle/src/EventListener/DataContainer/DefaultGlobalOperationsListener.php

```diff
@@ -109,7 +111,7 @@ private function getDefaults(string $table): array
             ];
         }
 
-        if ($canEdit || $canCopy || $canDelete) {
+        if ($canEdit || $canCopy || $canDelete || ($canSort && ($isParentMode || $isTreeMode || $isExtendedTreeMode))) {
```

Comment:
```suggestion
        if ($canEdit || $canCopy || $canDelete || $canSort) {
```

Is it really necessary to limit this to certain modes? BTW, mode 2 and 3 are also sortable.

Reply:
no, mode 2 and mode 3 are not sortable by an operation. They are sortable using the filter/sorting toolbar.
The check is necessary, otherwise we would render the edit-all operation for the other modes, even if all other operations are not allowed.